widget/cardcontainer.go

```diff
@@ -0,0 +1,220 @@
+package widget
+
+import (
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/internal/widget"
+	"fyne.io/fyne/theme"
+)
+
+// CardContainer widget groups title, subtitle with content and a header image
+type CardContainer struct {
+	BaseWidget
+	Title, SubTitle string
```

Comment:
nit: subtitle is one word, not a concatenation
```suggestion
	Title, Subtitle string
```